main.go

```diff
@@ -229,22 +243,34 @@ func read_reveals(conn *websocket.Conn, player_index int16, player_pos_index *in
 	}
 }
 
-func send_reveals(conn *websocket.Conn, player_index int16, player_pos_index *int32, afk *bool) {
+func send_reveals(conn *websocket.Conn, player_index int16, player_pos_index *int32, afk *bool, player_game_id *int32) {
 	ticker := time.NewTicker(33 * time.Millisecond)
 	defer ticker.Stop()
 	defer conn.Close()
 	var reveal_index = 0
 	var flag_index = 0
 	for {
 		<- ticker.C
+		
 		if (*afk) {
 			continue
 		}
+		message, err := conn.NextWriter(websocket.BinaryMessage)
+		if (*player_game_id != game_id) {
+			reveal_index = 0;
+			flag_index = 0;
+			*player_game_id = game_id
+			binary.Write(message, binary.BigEndian, int32(-1))
+			if err := message.Close(); err != nil {
+				return
+			}
+			continue
+		}
```

Comment:
If the player's game_id is out of sync, send a reset event (just -1) and reset the indices to 0